Formula/subversion.rb

```diff
@@ -76,7 +77,7 @@ def install
         APR=#{Formula["apr"].opt_prefix}
         APU=#{Formula["apr-util"].opt_prefix}
       ]
-      scons(*args)
+      system "scons", *args
```

Comment:
Could you also, as three separate commits in this same PR, do the same change in `subversion@1.8`, `libswiften` and `wesnoth`?